scripts/dump_wasm_and_state_for_all_chains.sh

```diff
@@ -9,9 +9,8 @@
 #
 # All arguments are optional.
 
-PARA_ID=${1:-2015}
```

Comment:
As noted in the previous pr, the collators do no longer support this flag.

Reply:
even better. one less source of inconsistency